### DISCUSSION TITLE
log.info in defaultErrorFormatter

### DISCUSSION THREAD

**Discussion by PieterJanVdb** [mercurius-js/mercurius#983] [Q&A] [answered]
I was wondering what the reasoning was for the following change (https://github.com/mercurius-js/mercurius/pull/675/commits/a115d81c010686eadd2ae68438c4c4a8ad8ab7d3#diff-0af7067b49efcaf7dae9a3707f2e79638b336e72780d2b834e007f80ef9a5fa0):

![image](https://user-images.githubusercontent.com/7528099/233096979-f27e4c5e-d3f2-472f-ab63-bfa19d2890d8.png)

Is it because a GraphQL error could represent anything from a server error to a client validation error and log.info is the lowest common denominator log level wise?

**Comment by mcollina** [comment, marked as answer]
Because an error in GraphQL will return a 200 anyway. It's an expected response and not an error situation for the server.

**Reply by PieterJanVdb** [reply]
Hmm out of curiosity, would the best thing to do be to trace any server errors that might still result in a 200 (eg some connection not resolving correctly) separately from w/e the defaultErrorFormatter is doing in eg. the onResolution hook and assume that all logging from the defaultErrorFormatter is informational, while the developer is responsible for separately tracing actual exceptions?

**Reply by mcollina** [reply]
if there are specicific class of errors you want to treat differentely, you can go ahead and implement your own formatter. I actually recommend to.
